com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/UndeployAppStep.java

```diff
@@ -126,6 +126,10 @@ private void deleteApplicationRoutes(List<CloudRoute> routes, String uri, CloudC
         getStepLogger().debug(Messages.ROUTE_DELETED, uri);
     }
 
+    private boolean isTcpPortRoute(String uri) {
```

Comment:
SystemParametersBuilder::isTcpOrTcpsRoute is a misleading name, I've change it to isTcpOrTcpsProtocol since it checks the protocol for the module and not the actual uri string